Detectors/ITSMFT/ITS/workflow/src/DCSParserSpec.cxx

```diff
@@ -363,16 +363,15 @@ void ITSDCSParser::pushToCCDB(EndOfStreamContext* ec)
   auto image = o2::ccdb::CcdbApi::createObjectImage(&mConfigDCS, &info);
   info.setFileName(filename);
 
-  if (ec) { // send to ccdb-populator wf only if there is an EndOfStreamContext
-    LOG(info) << "Class Name: " << class_name << " | File Name: " << filename
-              << "\nSending to ccdb-populator the object " << info.getPath() << "/" << info.getFileName()
-              << " of size " << image->size() << " bytes, valid for "
-              << info.getStartValidityTimestamp() << " : "
-              << info.getEndValidityTimestamp();
+  // Send to ccdb-populator wf
+  LOG(info) << "Class Name: " << class_name << " | File Name: " << filename
+            << "\nSending to ccdb-populator the object " << info.getPath() << "/" << info.getFileName()
+            << " of size " << image->size() << " bytes, valid for "
+            << info.getStartValidityTimestamp() << " : "
+            << info.getEndValidityTimestamp();
 
-    ec->outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBPayload, "DCS_CONFIG", 0}, *image);
-    ec->outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBWrapper, "DCS_CONFIG", 0}, info);
-  }
+  pc.outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBPayload, "DCS_CONFIG", 0}, *image);
+  pc.outputs().snapshot(Output{o2::calibration::Utils::gDataOriginCDBWrapper, "DCS_CONFIG", 0}, info);
 
   if (!mCcdbUrl.empty()) { // if url is specified, send object to ccdb from THIS wf
```

Comment:
I thought you want to send either to CCDBPopulator or to your locally defined server. Now you are sending unconditionally to the populator channels and optionally also to the test server. What is the point?

Reply:
Yes, good point, I will put this inside of an if/else statement.

Reply:
Added in https://github.com/AliceO2Group/AliceO2/pull/9533/commits/50a51eece57ecc55420cda7f553d36770d688b72